README.md

```diff
@@ -1,3 +1,6 @@
+# Uniswap Widgets
```

Comment:
Since this readme is the main doc on the npm page for the widgets package, I was thinking we should either have the widget section first, or limit this main `README.md` file to functioning as an index that points to `WIDGETS_README.md` and `INTERFACE_README.md`